pkg/generators/openapi_test.go

```diff
@@ -2358,10 +2358,6 @@ func TestMarkerComments(t *testing.T) {
 		package foo
 
 		// +k8s:openapi-gen=true
-		// +k8s:validation:maxProperties=10
-		// +k8s:validation:minProperties=1
-		// +k8s:validation:exclusiveMinimum
-		// +k8s:validation:exclusiveMaximum
```

Comment:
Sorry if this is obvious, but why are we removing these tags from this test input?

Reply:
added back